telegram/ext/dictpersistence.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+#
+# A library that provides a Python interface to the Telegram Bot API
+# Copyright (C) 2015-2018
+# Leandro Toledo de Souza <devs@python-telegram-bot.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains the DictPersistence class."""
+from telegram.utils.helpers import decode_user_chat_data_from_json,\
+    decode_conversations_from_json, enocde_conversations_to_json
+
+try:
+    import ujson as json
+except ImportError:
+    import json
+from collections import defaultdict
+from telegram.ext import BasePersistence
+
+
+class DictPersistence(BasePersistence):
```

Comment:
How is this class useful? Like it doesn't actually save anywhere does it?

Reply:
What class do you mean?

Reply:
DictPersistence
I don't really see the use of it, since it doesn't save anywhere, right? And therefore isn't persistent?

Reply:
It gives you easy acces to the dicts/json from the persistence class. So you can use a job to store them. It accepts json to restore them. @tsnoam requested thuis to be present

Reply:
Alright, so it's for more manual handling of persistent data. Got it. :)